<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Prop</class>
 <widget class="QWidget" name="Prop">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>330</width>
    <height>120</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>330</width>
    <height>120</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>330</width>
    <height>120</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="windowIcon">
   <iconset theme="icon.png">
    <normaloff>.</normaloff>.</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QWidget" name="widget_2" native="true">
     <layout class="QGridLayout" name="gridLayout_2">
      <property name="leftMargin">
       <number>5</number>
      </property>
      <property name="topMargin">
       <number>5</number>
      </property>
      <property name="rightMargin">
       <number>5</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="horizontalSpacing">
       <number>0</number>
      </property>
      <property name="verticalSpacing">
       <number>5</number>
      </property>
      <item row="1" column="1">
       <widget class="QLineEdit" name="sizeEdit">
        <property name="styleSheet">
         <string notr="true">border-style: outset;
border-width: 1px;
border-radius: 5px;
    border-color: beige;
    min-width: 10em;
    padding: 2px;
border-color: grey;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="NAME">
        <property name="text">
         <string>Имя</string>
        </property>
        <property name="buddy">
         <cstring>nameEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="SIZE">
        <property name="text">
         <string>Размер</string>
        </property>
        <property name="buddy">
         <cstring>sizeEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QLabel" name="sizeLabel">
        <property name="minimumSize">
         <size>
          <width>25</width>
          <height>0</height>
         </size>
        </property>
        <property name="layoutDirection">
         <enum>Qt::LeftToRight</enum>
        </property>
        <property name="text">
         <string>байт</string>
        </property>
        <property name="textFormat">
         <enum>Qt::RichText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLineEdit" name="nameEdit">
        <property name="styleSheet">
         <string notr="true">border-style: outset;
border-width: 1px;
border-radius: 5px;
    border-color: beige;
    min-width: 0em;
    padding: 1px;
border-color: grey;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="PATH">
        <property name="text">
         <string>Путь</string>
        </property>
        <property name="buddy">
         <cstring>pathEdit</cstring>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QLineEdit" name="pathEdit">
        <property name="styleSheet">
         <string notr="true">border-style: outset;
border-width: 1px;
border-radius: 5px;
    border-color: beige;
    min-width: 10em;
    padding: 2px;
border-color: grey;</string>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QWidget" name="widget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>10</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <property name="spacing">
       <number>0</number>
      </property>
      <item row="0" column="0">
       <spacer name="horizontalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>40</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="0" column="2">
       <widget class="QPushButton" name="closeButton">
        <property name="styleSheet">
         <string notr="true">border-style: outset;
border-width: 1px;
border-radius: 5px;
    border-color: beige;
    min-width: 7em;
    padding: 1px;
border-color: grey;</string>
        </property>
        <property name="text">
         <string>Close</string>
        </property>
        <property name="autoRepeat">
         <bool>false</bool>
        </property>
        <property name="autoDefault">
         <bool>false</bool>
        </property>
        <property name="default">
         <bool>false</bool>
        </property>
        <property name="flat">
         <bool>false</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
